package.json

```diff
@@ -2,12 +2,13 @@
   "name": "mapbox-gl-function",
   "version": "1.0.0",
   "description": "Evaluate a Mapbox GL style function",
-  "dependencies": {},
   "devDependencies": {
+    "eslint": "^0.22.1",
+    "mapbox-gl-style-spec": "git+https://github.com/mapbox/mapbox-gl-style-spec.git#v8-scales",
```

Comment:
I'd like to merge mapbox/mapbox-gl-style-spec#290 before we merge this PR so that we can use the slightly more stable `v8` branch as a dependency instead.